plugins/events-backend/src/service/http/HttpPostIngressEventPublisher.ts

```diff
@@ -92,8 +94,12 @@ export class HttpPostIngressEventPublisher implements EventPublisher {
     const path = `/${topic}`;
 
     router.post(path, async (request, response) => {
+      const requestDetails = {
+        body: request.body,
```

Comment:
Hm, how does this interact with express body handlers up the chain? For example, SHOULD one put a `.use(json())` on there to ensure that the body has been parsed and dealt with already? What if some providers' payloads are json and some other providers aren't? 🤔 Essentially, where lies the responsibility for handling the parsing of incoming data?

Reply:
yes, it is not the raw body and interacts with the `.use(json())`.

Also, the router created at `HttpPostIngressEventPublisher.createRouter` contains

```ts
    const router = Router();
    router.use(express.json());

    // [...]

    router.use(errorHandler());
```

Reply:
I added docs to the interface to clarify this legit question which may pop up at users of it in the same way.